File: app/src/main/java/audiolibros/example/com/audiolibros/Aplicacion.java
package audiolibros.example.com.audiolibros;

import android.app.Application;

import java.util.List;

/**
 * Created by William_ST on 03/02/19.
 */

public class Aplicacion extends Application {

    private List<Libro> listaLibros;
    private AdaptadorLibrosFiltro adaptador;

    @Override
    public void onCreate() {
        super.onCreate();
        listaLibros = Libro.ejemploLibros();
        adaptador = new AdaptadorLibrosFiltro(this, listaLibros);
    }

    public AdaptadorLibrosFiltro getAdaptador() {
        return adaptador;
    }

    public List<Libro> getListaLibros() {
        return listaLibros;
    }

}

File: app/src/main/java/audiolibros/example/com/audiolibros/Aplicacion.kt
package audiolibros.example.com.audiolibros

import android.app.Application
import android.graphics.Bitmap
import android.util.LruCache

import com.android.volley.RequestQueue
import com.android.volley.toolbox.ImageLoader
import com.android.volley.toolbox.Volley

import audiolibros.example.com.audiolibros.ejemploLibros

/**
 * Created by William_ST on 03/02/19.
 */

class Aplicacion : Application() {

    val listaLibros = ejemploLibros()
    lateinit var adaptador: AdaptadorLibrosFiltro
    lateinit var colaPeticiones : RequestQueue

    override fun onCreate() {
        super.onCreate()
        adaptador = AdaptadorLibrosFiltro(this, listaLibros)
        colaPeticiones = Volley.newRequestQueue(this)
    }

}